spec/system/searching/search_filters_and_results_spec.rb

```diff
@@ -22,7 +22,7 @@
     column_headings = page.first('.app-dashboard-table thead tr').text.squish
 
     expect(column_headings).to eq(
-      'Applicant Ref. no. Date received Date closed Caseworker Status'
+      'Applicants name Reference number Date received Date closed Caseworker Status'
```

Comment:
This is the search table, so revert for now (might be something we need to change in a later PR)